hpx/util/detail/basic_function.hpp

```diff
@@ -168,7 +168,7 @@ namespace hpx { namespace util { namespace detail
                     vtable::reconstruct<target_type>(&object, std::forward<F>(f));
                 } else {
                     reset();
-                    vtable::destruct<empty_function<R(Ts...)> >(&object);
+                    vtable::delete_<empty_function<R(Ts...)> >(&object);
```

Comment:
After looking through the report again, I think it is a false positive. Changing it to delete_ shouldn't do any harm and silences this warning, do you agree?